frontend/src/app/app.module.ts

```diff
@@ -420,7 +417,6 @@ export class OpenProjectModule implements DoBootstrap {
     registerCustomElement('opce-macro-wp-quickinfo', WorkPackageQuickinfoMacroComponent, { injector });
     registerCustomElement('opce-ckeditor-augmented-textarea', CkeditorAugmentedTextareaComponent, { injector });
     registerCustomElement('opce-draggable-autocompleter', DraggableAutocompleteComponent, { injector });
-    registerCustomElement('opce-attribute-help-text', AttributeHelpTextComponent, { injector });
     registerCustomElement('opce-static-attribute-help-text', StaticAttributeHelpTextComponent, { injector });
```

Comment:
I stumbled over that and thought at first, it could be replaced as well. But probably not, because it is not really an attribute help text but something that just looks like it..